### DISCUSSION TITLE
Reduce processing time

### DISCUSSION THREAD

**Discussion by mrNicky** [coqui-ai/TTS#452] [General]
If anyone, have any ideas on to reduce processing time. 20 secondes for me, between running command and saving wav output.
with a one line sentence.

I used default tts command like this:

` tts --text "blabla wha. . .." `

```
> tts_models/en/ljspeech/tacotron2-DDC is already downloaded.
 > vocoder_models/en/ljspeech/hifigan_v2 is already downloaded.
 > Using model: Tacotron2
 > Generator Model: hifigan_generator
Removing weight norm...
 > Text:  blablablablablablablablablablablablablablablablablablablablablablablablablablablablablabla
 > Text splitted to sentences.
 > Processing time: 8.45993447303772
 > Real-time factor: 0.6050886026393558
 > Saving output to tts_output.wav
```

**Comment by erogol** [comment]
how fast do you need? It is already faster than the real-time  (length of the generated audio is longer than the execution time) 
` > Real-time factor: 0.6050886026393558`

**Comment by L0CH1NBEK** [comment]
I also wondering, that is it possible to reduce processing time? the time between running command and getting the output .wav file.
